nixos/tests/musescore.nix

```diff
@@ -2,13 +2,15 @@ import ./make-test-python.nix ({ pkgs, ...} :
 
 let
   # Make sure we don't have to go through the startup tutorial
-  customMuseScoreConfig = pkgs.writeText "MuseScore3.ini" ''
+  customMuseScoreConfig = pkgs.writeText "MuseScore4.ini" ''
     [application]
-    startup\firstStart=false
+    hasCompletedFirstLaunchSetup=true
+
+    [project]
+    preferredScoreCreationMode=1
 
     [ui]
-    application\startup\showTours=false
-    application\startup\showStartCenter=false
+    skippedVersion=4.0.1
```

Comment:
This is no longer necessary since you updated the version to 4.0.1. I added this since at the time your branch was 4.0.0, so I had to skip the update prompt for the CI.

Reply:
However, I now realize that a step to skip that prompt *if* it appears might be necessary since, in the future, a new release will trigger the prompt.

Reply:
Will probably work fine for now though.

Reply:
While 4.0.1 was out the 4.0.0 build would display a 4.0.1 upgrade prompt. Disabling this is called "skipping" a version by musescore.

Reply:
> While 4.0.1 was out the 4.0.0 build would display a 4.0.1 upgrade prompt. Disabling this is called "skipping" a version by musescore.

I think that it doesn't make sense to use `pkgs.musescore.version`, or any other version.. Because we want to skip the next version upgrade prompt, and we can't know now in advance to which version there will be an update.. I removed this `[ui]` section. This test will fail when a new version will be available, which is a good indicator IMO.

Reply:
Thanks for remembering this!